modules/op-release-notes-1-12.adoc

```diff
@@ -188,3 +188,17 @@ You must provide these parameters using secrets. After upgrading to {pipelines-s
 * Before this update, if the GitHub API provided a paginated response, {pac} used only the first page of the response. With this update, all paginated responses are processed fully.
 
 * Before this update, the {tekton-chains} controller crashed when the host address of KMS Hashicorp Vault was configured incorrectly or when {tekton-chains} was unable to connect to the KMS Hashicorp Vault. With this update, {tekton-chains} logs the connection error and does not crash.
+
+[id="release-notes-1-12-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.12.1
+
+With this update, {pipelines-title} General Availability (GA) 1.12.1 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-12-1_{context}"]
+=== Fixed issues
+
+* Before this update, when using {pac}, after you pushed changes to a pull request, the CI status check for this pull request could remain as `waiting for status to be reported` and the associated pipeline run did not complete. With this update, after you push changes to a pull request, the associated pipeline run completes normally and the CI status check for the pull request is updated.
```

Comment:
where is this changelog come from? from which srvkp i mean?

Reply:
even i have the same question 
from which srvkp its coming :thinking:

Reply:
so yeah please @mramendi remove this changelog since it's wrong and use this instead (with better english/phrasing etc..):

when paac is configured with the custom console driver to output a custom console url it would in certain case crashes the controller, this should now be fixed

Reply:
@chmouel what is "crashed the controller" about, in user terms? I don't think the user directly interacts with the controller?

Reply:
ah okay, the status would never be updated then if we speak in user facing term..